code/restful-api/restful-api/Models/Ingresso.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+namespace RestfulApi.Models
+{
+    public class Ingresso
+    {
+        public long Id { get; set; }
+
+        public string NumeroDoIngresso { get; set; }
+
+        public string Nome { get; set; }
+
+        public string Sobrenome { get; set; }
+
+        public string Celular { get; set; }
+
+        public string Email { get; set; }
+
+        public Historico_Eventos Historico_EventosId { get; set; }
```

Comment:
As I said previously, I don't think this is needed.

Reply:
Retirado.